homeassistant/helpers/intent.py

```diff
@@ -221,12 +221,20 @@ async def async_handle(self, intent_obj: Intent) -> IntentResponse:
 
         response = intent_obj.create_response()
         response.async_set_speech(self.speech.format(state.name))
-        response.async_set_target(
-            IntentResponseTarget(
-                name=state.name,
-                type=IntentResponseTargetType.ENTITY,
-                id=state.entity_id,
-            )
+
+        # Targets go from general to specific
+        response.async_set_targets(
+            [
+                IntentResponseTarget(
```

Comment:
I don't think that this is a target: DOMAIN. It's just a single entity that is targeted. DOMAIN can only be applied on top of something else, like all lights in an area.